airflow/api_fastapi/parameters.py

```diff
@@ -144,17 +146,24 @@ def depends(self, dag_display_name_pattern: str | None = None) -> _DagDisplayNam
         return self.set_value(dag_display_name_pattern)
 
 
+# SortParam Implementations
 class SortParam(BaseParam[str]):
```

Comment:
@pierrejeambrun, I would like your take on the SortParam approach to make it generic to all model types. I thought this should be generic similar to `apply_sorting()`

Reply:
Note: I have tested the endpoints and they are working as expected.

Reply:
@bugraoz93 , I think, this should be on a separate PR. At least to get it merged ASAP as I'm working on list dag run and I too need this

Reply:
@rawwar, let's agree on a solution first for the SortParam. I am not spending too much time just copy paste the changes I am doing for DAG endpoints already. If this will block you more due to other things, I would love to cherry-pick those and create another PR. Thanks!